coalib/settings/Section.py

```diff
@@ -161,6 +164,11 @@ def __getitem__(self, item, ignore_defaults=False):
 
         res = self.contents.get(key, None)
         if res is not None:
+            if res.to_append and self.defaults and (res.key in self.defaults):
+                return Setting(
+                    key=key,
+                    value=self.defaults[key].value + ',' + res.value,
+                    origin=res.origin)
```

Comment:
hm I mentioned earlier that we might want to override the `.value` property getter to raise an exception if to_append is true because the value is likely wrong, then this will have to be changed!